code/game/objects/items/storage/fancy.dm

```diff
@@ -27,20 +27,19 @@
 	for(var/i = 1 to storage_slots)
 		new spawn_type(src)
 
-/obj/item/storage/fancy/update_icon(itemremoved = 0)
+/obj/item/storage/fancy/update_icon()

```

Comment:
if you're going to remove this snowflake arg can you update the callers to this proc as well

Reply:
I'm pretty sure there is hack in the parent storage types removal proc for this

Reply:
addressed thanks.